README.md

```diff
@@ -30,8 +30,10 @@
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/221178975-e5a9dc05-529a-4634-b3ba-12c155bbf6a4.png)

**[Optional]**

- [ ] It would be nice if you have your own logo then use it here otherwise remove this broken link to make your readme file more descriptive, professional and clean :+1: